File: CodeGenDotNet 159059-1372 (a)/VBCode/Chapter 1/HelloWorldAndSimpleClass/HelloWorldViaBruteForce.vb
Option Strict On
Option Explicit On

Imports System

'! Class Summary: Create Hello World program using direct stream output

Public Class HelloWorldViaBruteForce

#Region "Public Methods and Properties"
   Public Shared Sub GenerateOutput(ByVal outputFileName As String)
      Dim writer As IO.StreamWriter

      Try
         writer = New IO.StreamWriter(outputFileName)
         writer.WriteLine("Option Strict On")
         writer.WriteLine("Option Explicit On")
         writer.WriteLine("")
         writer.WriteLine("Imports System")
         writer.WriteLine("")
         writer.WriteLine("'! Class Summary: Hello World target output ")
         writer.WriteLine("")
         writer.WriteLine("Public Class TargetOutput")
         writer.WriteLine("")
         writer.WriteLine("#Region " & Chr(34) & "Public Methods and Properties" & Chr(34))
         writer.WriteLine("   Public Shared Sub Main()")
         writer.WriteLine("      Console.WriteLine(" & Chr(34) & "Hello World" & Chr(34) & ")")
         writer.WriteLine("   End Sub")
         writer.WriteLine("#End Region")
         writer.WriteLine("")
         writer.WriteLine("End Class")
      Finally
         If Not writer Is Nothing Then
            writer.Flush()
            writer.Close()
         End If
      End Try
   End Sub
#End Region

End Class

File: CodeGenDotNet 159059-1372 (a)/CSharp/Chapter 1/HelloWorldAndSimpleClass/HelloWorldViaBruteForce.cs
using System;

/// <summary>
/// Create Hello World program using direct stream output
/// </summary>
public class HelloWorldViaBruteForce
{
	#region Public Methods and Properties
	public static void GenerateOutput( string outputFileName )
	{
		System.IO.StreamWriter writer = null;
		try
		{
			writer = new System.IO.StreamWriter( outputFileName );
			writer.WriteLine( "using System;" );
			writer.WriteLine( "" );
			writer.WriteLine( "/// <summary>" );
			writer.WriteLine( "/// Hello World target output" );
			writer.WriteLine( "/// </summary>" );
			writer.WriteLine( "public class TargetHelloWorld" );
			writer.WriteLine( "{" );
			writer.WriteLine( "	#region Public Methods and Properties" );
			writer.WriteLine( "	public static void Main()" );
			writer.WriteLine( "	{" );
			writer.WriteLine( "		Console.WriteLine( \"Hello World\" );" );
			writer.WriteLine( "	}" );
			writer.WriteLine( "	#endregion" );
			writer.WriteLine( "}" );
		}
		finally
		{
			if( writer != null )
			{
				writer.Flush();
				writer.Close();
			}
		}
	}
	#endregion
}
